doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/managing-iot-data.md

```diff
@@ -1 +1,558 @@
+> db.devices.find();
+{ "_id" : ObjectId("57ed0f5b5755b6010061870b"), "protocol" : "UL20", "internalId" : null, "subservice" : "/Franchise1", "service" : "tourguide", "name" : "0115206c51f60b48b77e4c937835795c33bb953f", "type" : "Restaurant", "id" : "0115206c51f60b48b77e4c937835795c33bb953f-kitchen-temperature", "creationDate" : ISODate("2016-09-29T12:55:55.109Z"), "subscriptions" : [ ], "active" : [ { "type" : "Number", "name" : "temperature:kitchen", "object_id" : "t" } ], "__v" : 0 }
+```
+
+As we can see, the device has been registered with the IoT Agent.  There is one more check we can do, as we are adding the sensor to an existing entity in Orion.  We can do a request to orion for the restaurant and see if the new attribute has been added:
+```
+$ curl -s -H 'Fiware-Service: tourguide' http://localhost:1026/v2/entities/0115206c51f60b48b77e4c937835795c33bb953f | json_reformat
+{
+	"id": "0115206c51f60b48b77e4c937835795c33bb953f",
+	"type": "Restaurant",
+	"address": {
+    	"type": "PostalAddress",
+    	"value": {
+        	"streetAddress": "Cuesta de las Cabras Aldapa 2",
+        	"addressRegion": "Araba",
+        	"addressLocality": "Alegría-Dulantzi",
+        	"postalCode": "01240"
+    	},
+    	"metadata": {
+
+    	}
+	},
+	"aggregateRating": {
+    	"type": "AggregateRating",
+    	"value": {
+        	"ratingValue": 3,
+        	"reviewCount": 64
+    	},
+    	"metadata": {
+
+    	}
+	},
+	"capacity": {
+    	"type": "PropertyValue",
+    	"value": 200,
+    	"metadata": {
+
+    	}
+	},
+	"department": {
+    	"type": "Text",
+    	"value": "Franchise1",
+    	"metadata": {
+
+    	}
+	},
+	"description": {
+    	"type": "Text",
+    	"value": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además de su menú del día y carta, también ofrece menú de sidrería. El menú del día cuesta 9 euros. Los fines de semana la especialidad de la casa son las alubias con sacramentos. En lo que a bebidas se refiere, hay una amplia selección además de la sidra. Cabe destacar que se puede hacer txotx. La capacidad del establecimiento es de 50 personas pero la sidrería no dispone de aparcamiento.%5cn%5cnHORARIO: %5cn%5cnLunes a domingo: 9:00-17:00 y 19:00-23:00.",
+    	"metadata": {
+
+    	}
+	},
+	"location": {
+    	"type": "geo:point",
+    	"value": "42.8404625, -2.5123277",
+    	"metadata": {
+
+    	}
+	},
+	"name": {
+    	"type": "Text",
+    	"value": "Elizalde",
+    	"metadata": {
+
+    	}
+	},
+	"occupancyLevels": {
+    	"type": "PropertyValue",
+    	"value": 0,
+    	"metadata": {
+        	"timestamp": {
+            	"type": "DateTime",
+            	"value": "2016-09-29T14:19:39.267Z"
+        	}
+    	}
+	},
+	"priceRange": {
+    	"type": "Number",
+    	"value": 0,
+    	"metadata": {
+
+    	}
+	},
+	"telephone": {
+    	"type": "Text",
+    	"value": "945 400 868",
+    	"metadata": {
+
+    	}
+	},
+	"temperature:kitchen": {
+    	"type": "Number",
+    	"value": " ",
+    	"metadata": {
+
+    	}
+	}
```

Comment:
I think we can be here more compact if we only request the referred attribute or only show an excerpt of the json entity representation